js/common/lib/tensor.ts

```diff
@@ -233,6 +233,29 @@ export interface TensorConstructor {
   new(data: Tensor.DataType, dims?: readonly number[]): Tensor;
   // #endregion
 }
+export interface TensorFactory {
+  /**
+   * Create a new tensor object from image object
+   * @param image - Image object holding the image data
```

Comment:
please describe more details ( make differences to the other override for input type HTMLImageElement)

add information on how to create/get the ImageData object, and any restrictions.

eg.

- see also: (link to MDN) https://developer.mozilla.org/en-US/docs/Web/API/ImageData
- restrictions: color space "display-p3" is not supported.